### ISSUE TITLE
Integration Guide?

### ISSUE THREAD

**Issue by billiegoose** [iefserge/eshttp#14]
I'm thinking about making a backend for this module for my [`dukboot`](https://gitlab.com/wmhilton/dukboot) runtime. I didn't see any instructions on how to actually write backends though. Do I use `eshttp` as a dependency and write a wrapper module?

**Comment by iefserge** [comment]
Yep, that should work. See example of the runtime.js backend https://github.com/iefserge/eshttp/blob/master/index-runtime.js
It calls `setBackend` to provide backend object, then re-exports eshttp.

Backend itself is a set of functions for working with sockets on the platform https://github.com/iefserge/eshttp/blob/master/backend/backend-runtime.js

Feel free to open an issue about backend API improvements if it makes sense.
